code/modules/orbit/orbit.dm

```diff
@@ -54,7 +52,10 @@
 	orbiter.loc = targetloc
 	orbiter.update_parallax_contents()
 	lastloc = orbiter.loc
-
+	if(orbiter.orbiters)

```

Comment:
this if check is redundant and unnecessary; `for(var/I in null)` works correctly and does nothing